docs/conf.py

```diff
@@ -191,3 +192,11 @@ def setup(app):
 (common_docs_path / "craft-parts").symlink_to(
     craft_parts_docs_path, target_is_directory=True
 )
+
+# By default, the documentation includes a feedback button at the top.
+# You can disable it by setting the following configuration to True.
+disable_feedback_button = False
```

Comment:
Is this part necessary? I tried it locally with only defining `"github_issues"` above and the feedback button seems to show up and work fine

Reply:
This is boilerplate we've been sharing between repos. Refactored: [bab71e8](https://github.com/canonical/rockcraft/pull/806/commits/bab71e83ee32cddad779d110417e55b82cf5d10b)